storage/Cargo.toml

```diff
@@ -13,7 +13,7 @@ hex = "0.4.3"
 serde = { version = "1.0.199", features = ["derive"] }
 smallvec = { version = "1.13.2", features = ["serde", "write", "union"] }
 sha2 = "0.10.8"
-bytemuck = { version = "1.14.3", features = ["derive"] }
+integer-encoding = "4.0.0"
```

Comment:
Looks like you left this guy

Reply:
Yeah, we'll need it very soon as the one user (hashednode.rs) will be moving to storage very soon.